packages/celotool/src/e2e-tests/exit_test.ts

```diff
@@ -67,6 +68,13 @@ describe('exit tests', function(this: any) {
   }
 
   describe('when running a node', () => {
+    it('block limit should have been set using governance', async () => {
```

Comment:
I think this test is giving us false confidence. Does celo-blockchain actually read this parameter from the smart contract? Or is it just a coincidence that the value in celo-blockchain happens to be what we're setting the block gas limit to in the smart contract